docs/getting-started/installing/installing-core-linux.md

```diff
@@ -122,17 +74,50 @@ The next thing you will need is a `project.json` file that will outline the depe
 
 ## Run your App
 
-You need to restore packages for your app, based on your project.json, with `dnu restore`.
+You need to restore packages for your app, based on your project.json, with `dotnet restore`.
 
 ```console
-dnu restore
-dnx run
+dotnet restore
```

Comment:
Nit: 

Adding "$" (or other common symbol) makes it a bit more clear that this is exactly what you have to type in the command line.